File: src/main/java/io/github/bot/terminal/application/domains/integrations/slack/models/Channel.java
package io.github.bot.terminal.application.domains.integrations.slack.models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;
import lombok.experimental.Accessors;

@Data
@Accessors(chain = true)
@JsonIgnoreProperties(ignoreUnknown = true)
public class Channel {

    @JsonProperty("id")
    private String id;

    @JsonProperty("name")
    private String name;


}

File: src/main/kotlin/io/github/bot/terminal/application/domains/integrations/slack/models/Channel.kt
package io.github.bot.terminal.application.domains.integrations.slack.models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
class Channel {
    @JsonProperty("id")
    var id: String = ""

    @JsonProperty("name")
    var name: String = ""
}